source/simulator/assembly.cc

```diff
@@ -936,6 +942,11 @@ namespace aspect
     for (unsigned int q=0; q<n_q_points; ++q)
       for (unsigned int c=0; c<parameters.n_compositional_fields; ++c)
         material_model_inputs.composition[q][c] = composition_values[c][q];
+
+    const typename Triangulation<dim>::active_cell_iterator cell = input_finite_element_values.get_cell();
+    const typename DoFHandler<dim>::active_cell_iterator dof_cell(&triangulation,cell->level(),cell->index(),&dof_handler);
```

Comment:
The list of arguments to the constructor here could use a couple of newlines :-)